dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OpenaiChatController < ApplicationController
```

Comment:
I will add in a follow-up PR because I'd like to consult with @thomasoniii before adding a comment here. Thanks for the suggestion!

Reply:
I think of this thing as being pretty generic, and there's nothing in that class that looks SL specific. So I'd say keep it as is and it can be up for grab for anyone.

As we come up with SL/TT/etc specific functionality, we should subclass off of it for the specifics, though.

Reply:
Sounds good to me! Knowing that, I agree that the classname sounds general enough to the OpenAI chat completion API, so I guess it would only need clarifying if you meant "chat" to be specific somehow to EduBot